core/src/avm2/globals/flash/globalization/CurrencyFormatter.as

```diff
@@ -0,0 +1,9 @@
+package flash.globalization {
+    import __ruffle__.stub_constructor;
+
+    public dynamic class CurrencyFormatter {
```

Comment:
I think this should be a `final` class, not a `dynamic` class.

Reply:
Thanks I wasn't aware of difference before looking this up.